source/NVDAState.py

```diff
@@ -89,6 +90,43 @@ def updateCheckStateFile(self) -> str:
 	def guiStateFile(self) -> str:
 		return os.path.join(self.configDir, "guiState.ini")
 
+	@property
+	def defaultStartMenuFolder(self) -> str:
+		"""Name of a specific folder in the start menu, not a full path"""
+		return buildVersion.name
+
+	@cached_property
+	def startMenuFolder(self) -> str | None:
```

Comment:
Is there actually a point to save this in the registry, since it isn't configurable?

Reply:
I don't think this function saves anything to the registry, `startMenuFolder` is mostly used to `_createShortcutWithFallback`. This PR is mostly for code clean up, I would encourage a separate issue for any design changes